### PR TITLE
LPS-184711 Zendesk JWT integration

### PR DESCRIPTION
**Pull request by Odrakir1**

### CODE REVIEW

**Comment by Odrakir1** [comment]
ci:test:sf

**Comment by nNilton** [comment]
ci:test:sf

File: modules/apps/click-to-chat/click-to-chat-web/src/main/java/com/liferay/click/to/chat/web/internal/configuration/ClickToChatConfiguration.java [hunk 1]
```diff
@@ -43,4 +43,8 @@ public interface ClickToChatConfiguration {
 
 	public String siteSettingsStrategy();
 
+	public String chatProviderKeyId();
+
+	public String chatProviderSecretKey();
+
```
Comment (nNilton) [inline modules/apps/click-to-chat/click-to-chat-web/src/main/java/com/liferay/click/to/chat/web/internal/configuration/ClickToChatConfiguration.java:49]: sort alphabetically by method name

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by nNilton** [comment]
ci:test:sf

**Comment by nNilton** [comment]
ci:test:sf

**Review by nNilton — APPROVED** [review]

**Comment by nNilton** [comment]
Could you please look at this @joseabelenda ?

**Comment by joseabelenda** [comment]
resent:https://github.com/pt-liferay-solutions/liferay-portal/pull/4087

**Pull request closed without merging**